src/cloud-api-adaptor/test/e2e/common.go

```diff
@@ -28,6 +29,61 @@ const BUSYBOX_IMAGE = "quay.io/prometheus/busybox:latest"
 const WAIT_DEPLOYMENT_AVAILABLE_TIMEOUT = time.Second * 180
 const DEFAULT_AUTH_SECRET = "auth-json-secret-default"
 
+var testInitdata string = `algorithm = "sha384"
+version = "0.1.0"
+
+[data]
+"aa.toml" = '''
+[token_configs]
+[token_configs.coco_as]
+url = '%s'
+
+[token_configs.kbs]
+url = '%s'
+'''
+
+"cdh.toml"  = '''
+socket = 'unix:///run/confidential-containers/cdh.sock'
+credentials = []
+
+[kbc]
+name = 'cc_kbc'
+url = '%s'
+'''
+
+"policy.rego" = '''
+package agent_policy
+
+import future.keywords.in
+import future.keywords.every
+
+import input
+
+# Default values, returned by OPA when rules cannot be evaluated to true.
+default CopyFileRequest := false
```

Comment:
What was the reason behind this policy file being restrictive? does disabling `CopyFileRequest` prevent things like config map/secrets that we might want to use in other tests?

Reply:
Oh, I saw in a later commit that this was reverted, could you fixup up that commit into this one please for clarity?

Reply:
Yes, it's used for test case run correctly. I'll tidy the commits. thanks! @stevenhorsman